api/pkg/s3/routbucketcreate.go

```diff
@@ -0,0 +1,52 @@
+// Copyright 2019 The OpenSDS Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
```

Comment:
already there is routebucketput

Reply:
Yes, there is API routebucketput which basically responsible for creating folder inside bucket.